WordPress/src/main/res/layout/post_cardview_skeleton.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<android.support.v7.widget.CardView
```

Comment:
We need to improve this skeleton layout and have a separate issue for it ##8475.